src/client.rs

```diff
@@ -1354,7 +1354,7 @@ where
                     // Sync
                     // Frontend (client) is asking for the query result now.
                     'S' => {
-                        debug!("Sending query to server");
+                        debug!("Sending query to server  (in Sync mode)");
```

Comment:
No such thing as "sync" mode, this is the extended protocol. Maybe make the debug line something like:

```
Sending query to server with extended protocol
```

Reply:
Reverted this.